src/vs/workbench/browser/parts/editor/editorPart.ts

```diff
@@ -1274,14 +1274,20 @@ export class EditorPart extends Part implements IEditorPart, IEditorGroupService
 
 		// Respect option to reveal an editor if it is already visible
 		if (options && options.revealIfVisible) {
-			const editorsToCheck: BaseEditor[] = [];
-			if (activeEditor) { editorsToCheck.push(activeEditor); }
-			visibleEditors.forEach(e => { if (e !== activeEditor) { editorsToCheck.push(e); } });
-			for (let i = 0; i < editorsToCheck.length; i++) {
-				const editorToCheck = editorsToCheck[i];
-				if (input.matches(editorToCheck.input)) {
-					return editorToCheck.position;
-				}
+			const group = this.stacks.findGroup(input, true);
+			if (null !== group) {
+				return this.stacks.positionOfGroup(group);
+			}
+		}
+
+		const config = this.configurationService.getConfiguration<IWorkbenchEditorConfiguration>();
```

Comment:
Instead of getting the config each time we open an editor I suggest to store the value in a field of `EditorPart`. You can see in the [constructor](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/browser/parts/editor/editorPart.ts#L145) that we resolve configuration and we also react to [updates](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/browser/parts/editor/editorPart.ts#L173).